### ISSUE TITLE
Magepack bundle - Infinite Ajax Scroll jQuery is not defined

### ISSUE THREAD

**Issue by Rohaty** [magesuite/magepack#102]
Hi,
in my theme for eshop Magento2 I use for category Ajax infinity scroll. Everything works properly, shows no errors.
However, if I want to use the magepack bundle, when I open the category page, I get a "jQuery is not defined" error and it refers to bundle-category.js.
If I throw jquery-ias.min and ajaxinfinitescroll.js from magepack.config.js when creating a magepack bundle, then the error does not appear. Obviously, the error in the magepack bundle is related to infinityscroll. Where could be the problem?

If I use Requirejs, the error will not appear. I would like to use a magepack bundle. Can you advise me where the problem is?

**Comment by Vigneshmuthusamy** [comment]
Same error for me as well when using infinite scroll. We use weltpixel infinite scroll module

**Comment by Quazz** [comment]
Imo, this happens when developers don't properly define dependencies.

More than likely the infinitescroll file just assumes that jQuery will be available, rather than defining it as a dependency. 

So, whoever the dev is of that module should fix it.

**Comment by Vigneshmuthusamy** [comment]
Yeah. you're right. I fixed the plugin dependency and it worked. Whoever has this problem, please use the latest version of jquery ias plugin which would solve the issue. Most modules uses old version of ias module. This can be closed
